modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -329,6 +330,66 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_timeline(array &$variables)
   $variables['hide_from'] = $paragraph->get('field_oe_timeline_expand')->value;
 }
 
+/**
+ * Implements hook_preprocess_paragraph() for paragraph--oe-content-row--variant-inpage-navigation.html.twig.
+ */
+function oe_bootstrap_theme_preprocess_paragraph__oe_content_row__variant_inpage_navigation(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+
+  if (!$paragraph->get('field_oe_title')->isEmpty()) {
+    $variables['title'] = $paragraph->get('field_oe_title')->value;
+  }
+
+  $variables['full_layout'] = TRUE;
+  $variables['scrollspy_id'] = 'bcl-inpage-navigation';
+
+  $links = [];
+  $content = [];
+
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $delta => $sub_paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+    if ($sub_paragraph->hasField('field_oe_title') && !$sub_paragraph->get('field_oe_title')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_title')->first()->value;
+    }
+    elseif ($sub_paragraph->hasField('field_oe_text') && !$sub_paragraph->get('field_oe_text')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_text')->first()->value;
```

Comment:
you have to explain this one to me, why do we use these backups ?

Reply:
This are no backups, we manage three kind of paragraphs that can be added to a OE content row paragraph:
* Paragraphs which include a field called field_oe_title, then the title must be used at the inpage menu at the left part of the Content Row, and set an anchor for each title.
* Paragraphs which include a field field_oe_text which is used as a title for that paragraph type. Then we use that to mount the inpage navigation.
* Paragraph accordion made of n paragraph accordion items. Paragraph accordion per se doesn't have title, so we must use the title of the first Paragraph accordion item to use it to mount the navigation.